hazelcast-enterprise-kubernetes/README.md

```diff
@@ -161,9 +161,9 @@ spec:
         ports:
         - containerPort: 5701
         livenessProbe:
-          exec:
-            command:
-            - "./liveness.sh"
+          httpGet:
+            path: /hazelcast/health/node-state
+            port: 5701
```

Comment:
What do you think about putting a sample for readiness?

Reply:
hmm... I'd wait until we decide if we recommend using readiness at all (as discussed in the OpenShift TDD).